### ISSUE TITLE
Can't handle regex path definitions

### ISSUE THREAD

**Issue by BeiNacht** [instana/nodejs-sensor#140]
https://github.com/instana/nodejs-sensor/blob/626ab3c8258d4e91d42a61d79603532a921b35b4/packages/core/src/tracing/instrumentation/frameworks/koa.js#L79

We use a regex path definition in the koa router and instana throws an error for every call. So we can't start our service.

```
TypeError: b.path.endsWith is not a function
  | at byMostSpecificLayer (/app/node_modules/instana-nodejs-sensor/src/tracing/instrumentation/frameworks/koa.js:79:22)
  | at Array.sort (native)
  | at /app/node_modules/instana-nodejs-sensor/src/tracing/instrumentation/frameworks/koa.js:50:30
  | at process._tickCallback (internal/process/next_tick.js:68:7)
 ```

```router.all(/.*/, ....);```

**Comment by basti1302** [comment]
Thanks for the report, I'll take a look at this immediately.

**Comment by basti1302** [comment]
A fix has landed in version `1.67.1`.
